src/main/java/org/janelia/saalfeldlab/n5/GzipCompression.java

```diff
@@ -75,7 +75,7 @@ public InputStream getInputStream(final InputStream in) throws IOException {
 		if (useZlib) {
 			return new InflaterInputStream(in);
 		} else {
-			return new GzipCompressorInputStream(in);
+			return new GzipCompressorInputStream(in,true);
```

Comment:
Please give me the space and I accept :).

Reply:
space added!